swagger.yaml

```diff
@@ -124,6 +124,136 @@ paths:
           description: "Internal Server Error"
           schema:
             $ref: "#/definitions/500Response"
+  /user:
+    post:
+      tags:
+        - User
+      summary: Register user into system
+      parameters:
+        - name: user
+          in: body
+          description: Create user
+          schema:
+            $ref: "#/definitions/user"
+      responses:
+        "201":
+          description: OK
+          schema:
+            $ref: "#/definitions/user"
+        "500":
+          description: "Internal Server Error"
+          schema:
+            $ref: "#/definitions/500Response"
+  /user/reset-password:
```

Comment:
This should be actually in Forgot Password API.
Reset Password API should accept Current Password and New Password. if the current Password is valid then update new password.